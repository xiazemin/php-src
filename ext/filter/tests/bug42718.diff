004+ 1\000
004- 1