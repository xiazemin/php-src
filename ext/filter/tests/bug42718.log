
---- EXPECTED OUTPUT
unsafe_raw
4
bool(true)
1
---- ACTUAL OUTPUT
unsafe_raw
4
bool(true)
1\000
---- FAILED
