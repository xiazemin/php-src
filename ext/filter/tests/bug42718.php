<?php
echo ini_get('filter.default') . "\n";
echo ini_get('filter.default_flags') . "\n";
var_dump(FILTER_FLAG_STRIP_LOW == 4);
echo addcslashes($_GET['a'],"\0") . "\n";
?>
