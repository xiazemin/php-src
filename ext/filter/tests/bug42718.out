unsafe_raw
4
bool(true)
1\000